lib/irb.rb

```diff
@@ -698,45 +698,6 @@
 # and <tt>$DEBUG</tt> to +true+;
 # these have no effect on \IRB output.
 #
-# === Tracer
```

Comment:
Tracer integration needs some rework: https://github.com/ruby/irb/issues/752

I'll add the documentation back once that's completed.